src/wwwroot/controllers/DomainManagerCtrl.js

```diff
@@ -23,6 +23,7 @@
       var vm = this;
       vm.regexDomain = "(?=^.{4,253}$)(^((?!-)[a-zA-Z0-9-]{0,62}[a-zA-Z0-9]\\.)+[a-zA-Z]{2,63}$)";
       vm.activationPromise = activate();
+      vm.statusList = [{ name: $translate.instant('default_text'), id : 1}, {name: $translate.instant('enable_text'), id : 2}, {name: $translate.instant('disabled_text'), id : 3}];
```

Comment:
Will it be updated when user change language? (I know that by the moment we do not allow that, but anyway...